### ISSUE TITLE
All fields get cleared on Invite List validation error

### ISSUE THREAD

**Issue by ctrance** [universityofcalifornia/NeXt#406]
@joshuaselsky @shaun965 - On Events page, I filled in all the fields but have invalid Invite List value (emails were separated by semi-colon). After saving the event I got the email validation error but the whole form got cleared, I have to re-type everything. 

I think we should not clear the form whenever there's a validation error. What do you guys think?

**Comment by laurencullen** [comment]
I agree

**Comment by joshuaselsky** [comment]
Yes, definitely, especially in this case where the user may have typed a long list.

**Comment by ctrance** [comment]
@shaun965 @joshuaselsky - The uploaded image disappears, all other fields did not clear though.
